<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;监测功能界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>单次功能测试</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>271</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QProgressBar" name="progressBar_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QProgressBar" name="progressBar_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QProgressBar" name="progressBar_7">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QProgressBar" name="progressBar_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QProgressBar" name="progressBar_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QProgressBar" name="progressBar_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QProgressBar" name="progressBar_111">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 117, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="progressBar_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QProgressBar" name="progressBar_10">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="progressBar_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QProgressBar" name="progressBar_1">
       <property name="styleSheet">
        <string notr="true">color: rgb(99, 78, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>电池1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>电池2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>电池3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>电池4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>电池5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>电池6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>电池7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>电池8</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>电池9</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>电池10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>总电池</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>51</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="10" column="0">
      <widget class="QLineEdit" name="v1_10"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="v1_6"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="v1_2"/>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="v1_8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="v1_4"/>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="v1_7"/>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="v1_9"/>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="v1_111"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="v1_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="v1_5"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="v1_1">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电池1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>51</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="10" column="0">
      <widget class="QLineEdit" name="v2_10"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="v2_6"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="v2_2"/>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="v2_8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="v2_4"/>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="v2_7"/>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="v2_9"/>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="v2_111"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="v2_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="v2_5"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="v2_1">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电池1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>51</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="10" column="0">
      <widget class="QLineEdit" name="v3_10"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="v3_6"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="v3_2"/>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="v3_8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="v3_4"/>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="v3_7"/>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="v3_9"/>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="v3_111"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="v3_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="v3_5"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="v3_1">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电池1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>真实值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监测值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>误差值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>89</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC异常</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>390</y>
      <width>101</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>测试结果</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>误差阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>参考电压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;电压检测功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>891</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存测试结果</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>901</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>生成测试报告</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>41</width>
      <height>461</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>530</y>
      <width>121</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>95</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>ADC1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>ADC2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>ADC3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>ADC4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>ADC5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>ADC6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>ADC7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>ADC8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>ADC9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>ADC10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>ADC0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>640</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>760</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;电流检测功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;温度检测功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>真实值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>780</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>检测值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>780</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>780</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>660</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>误差值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>640</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>误差阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>730</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>单次功能测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>770</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存测试结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>770</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>770</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>770</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存测试结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>730</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>单次功能测试</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>误差值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>660</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>600</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>640</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>误差阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>610</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>真实值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>730</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>检测值</string>
    </property>
   </widget>
   <widget class="AnalogGaugeWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="AnalogGaugeWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>监测功能界面</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogGaugeWidget</class>
   <extends>QWidget</extends>
   <header location="global">analoggaugewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
